### PR TITLE
COMMERCE-6055 REPLACEMENT

### PR DESCRIPTION
**Pull request by garofalog**
https://issues.liferay.com/browse/COMMERCE-6055

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-subscription-type-web/src/main/resources/META-INF/resources/js/yearly/view.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+
+export default function ({modeExactdayOfYear, namespace}) {
+	function changeYearlyDeliveryCPSubscriptionTypeSettingsMode() {
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/commerce/commerce-product-subscription-type-web/src/main/resources/META-INF/resources/js/yearly/view.js:17]: Moving a function here will make it uncallable from the outside (have a look at its closure).

Also, the function should be invoked via event listener after removing:
`onChange='<%= "event.preventDefault(); changeYearlyDeliveryCPSubscriptionTypeSettingsMode();"`

File: modules/apps/commerce/commerce-product-subscription-type-web/src/main/resources/META-INF/resources/js/montly/view.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+
+export default function ({modeExactDayOfMonth, namespace}) {
+	function changeYearlyDeliveryCPSubscriptionTypeSettingsMode() {
```
Comment (FabioDiegoMastrorilli) [inline modules/apps/commerce/commerce-product-subscription-type-web/src/main/resources/META-INF/resources/js/montly/view.js:19]: Moving a function here will make it uncallable from the outside (have a look at its closure).

Also, the function should be invoked via event listener after removing:
`onChange='<%= "event.preventDefault(); changeYearlyDeliveryCPSubscriptionTypeSettingsMode();" %>`

**Review by FabioDiegoMastrorilli — CHANGES_REQUESTED** [review]

**Comment by FabioDiegoMastrorilli** [comment]
@garofalog please after testing also run `sf`

**Pull request closed without merging**